CMakeLists.txt

```diff
@@ -1,8 +1,13 @@
 CMAKE_MINIMUM_REQUIRED (VERSION 2.8.6)
 PROJECT (mqtt_cpp)
 
-OPTION(BUILD_EXAMPLES "enable building example applications" ON)
-OPTION(BUILD_TESTS "enable building test applications" ON)
+OPTION(BUILD_EXAMPLES "Enable building example applications" ON)
+OPTION(BUILD_TESTS "Enable building test applications" ON)
+OPTION(MQTT_NO_TLS "Disable building TLS code" ON)
+OPTION(MQTT_USE_WS "Enable building WebSockets code" OFF)
+OPTION(MQTT_USE_STR_CHECK "Enable UTF8 String check" ON)
+OPTION(MQTT_STD_VARIANT "Use std::variant from C++17 instead of boost::varient" OFF)
```

Comment:
I found a typo. `s/varient/variant/`. Could you fix this?

Reply:
Should be fixed now.

Reply:
Ah, I did the force push wrong.

*now* it should be fixed.